src/components/GalleryCarousel/GalleryCarousel.tsx

```diff
@@ -6,40 +6,62 @@ import {
   CarouselPrevious,
 } from "@/components/ui/carousel";
 import Image from "next/image";
+import Link from "next/link";
 
-const GalleryCarousel = ({ album }: { album: any[] }) => {
+const GalleryCarousel = ({
+  album,
+  length,
+}: {
+  album: any[];
+  length: string;
+}) => {
   return (
-    <section className="flex items-center justify-center w-full px-6 py-4 lg:px-20">
-      <div className="flex flex-col items-left w-full max-w-[1440px]">
-        <h2 className="text-green-900 text-2xl font-semibold">Galeria</h2>
-        <div className="flex justify-center">
-          <Carousel
-            opts={{
-              align: "start",
-              loop: true,
-            }}
-            className="flex flex-col gap-4 content-carousel py-4"
-          >
-            <CarouselContent className="-ml-0">
-              {album.map((photo: any, index: number) => (
-                <CarouselItem
-                  key={index}
-                  className="basis-1/1 md:basis-1/2 lg:basis-1/3 xl:basis-1/4"
-                >
+    <section className="flex items-center justify-center w-full">
+      <div className="flex justify-center w-full max-w-[1440px]">
+        <Carousel
+          opts={{
+            align: "start",
+            loop: true,
+          }}
```

Comment:
```suggestion
          }}
          plugins={[
            Autoplay({
              delay: 5000,
            }),
          ]}
```

We could add a autoPlay here